api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -527,9 +521,7 @@ private Workspace updateWorkspaceImpl(
           clonedRoles.put(entry.getKey(), WorkspaceAccessLevel.OWNER);
         }
       }
-      dbWorkspace =
-          workspaceAuthService.updateWorkspaceAcls(
-              dbWorkspace, clonedRoles, getRegisteredUserDomainEmail());
+      dbWorkspace = workspaceAuthService.updateWorkspaceAcls(dbWorkspace, clonedRoles);
```

Comment:
Moved this logic to the service impls because:
1. we prefer logic to reside in Services over Controllers
2. we can derive this from the dbWorkspace
3. we must choose the correct tier rather than this fixed value

Reply:
Thanks for the commentary! Otherwise I would have been squinting at this line.